### DISCUSSION TITLE
Any Idea for how to Tunnel all Outline Traffic to another server for Internet ?

### DISCUSSION THREAD

**Discussion by SniperTomb** [outline/outline#4214] [Self Hosting]
Hi

I have censored national network so I want to use outline for this
I don't have uncensored access to international network!

I have a **VPS1** in my country that have access to international network and second **VPS2** in Europe that have all access to network

how I can setup outline on my First VPS1 with Outline then tunnel it to VPS2 to access uncensored and wide network ?
Is there any way to setup 2 outline service on my VPS1 then tunnel all traffics it to VPS2 & VPS3 on different ports ?

Something like this picture but maybe with two Vps outside my country
![image](https://user-images.githubusercontent.com/114521061/193490095-57ddeb50-532f-4f91-8bfb-c9f846e67ee7.png)

**Comment by Limezy** [comment]
I don't really understand why or how Outline could be useful in your scenario ?
Do you have special needs such as a Youtube video frame inserted into one of your documents ?

**Reply by SniperTomb** [reply]
> I don't really understand why or how Outline could be useful in your scenario ? Do you have special needs such as a Youtube video frame inserted into one of your documents ?

every VPN protocol is blocked!
and I can't even connect to google.com
I just have connection to my country VPS and that have connection to network but with censored permissions ,
I need to connect to my VPS with outline and the VPS connect to another VPS in Europe

**Reply by tommoor** [reply]
https://github.com/Jigsaw-Code/?q=outline
